### ISSUE TITLE
kde: theme is partially applied

### ISSUE THREAD

**Issue by ca5ua1** [danth/stylix#149]
'kdeglobals' file is not created by stylix thou I specifically activate KDE target and no kdeglobals file exist in ~/.config/.

```
~ ➤ cd .config
.config ➤ fd kdeglobals
```

home-manager (casual/default.nix):
```
home-manager.users.casual = { pkgs,config, lib, ... }: {
stylix.autoEnable = true;
...
  stylix.targets.kde.enable = true;
...
```

DE: Hyprland
Full config at https://git.sual.in/casual/nixos-devices

**Comment by danth** [comment]
Some programs assume that the config file is writeable, so to avoid causing problems Stylix creates a separate copy under `$XDG_CONFIG_DIRS`. This should be picked up and merged with any settings from `~/.config/kdeglobals` if that exists.

**Comment by Casul51** [comment]
Yes, I see. Indeed there is stylix's path in `$XDG_CONFIG_DIRS` my problem that Dolphin partially apply theme, and KeePassXC doesn't apply at all. + for some reason Dolphin which run by Hyprland or Rofi doesn't apply theme at all (left). But if I run from console it partially get applied (right). Other apps seems work better but still in some apps there is white parts of the interface which I believe they shouldn't be

![image](https://github.com/danth/stylix/assets/33811764/110807b9-bef0-4118-95ff-c280ab2f358a)

**Comment by danth** [comment]
Hopefully #142 will fix this.

**Comment by abiriadev** [comment]
For those who searched for a partial theme issue like this: try enabling both `targets.kde` and `targets.qt` (if you set `autoEnable = false`). Dolphin(and some other KDE apps)'s theme will be affected by two different sources, KDE and Qt. Don't forget to remove any Qt-related configs in your existing HM config.

**Comment by trueNAHO** [comment]
> For those who searched for a partial theme issue like this: try enabling both `targets.kde` and `targets.qt` (if you set `autoEnable = false`). Dolphin(and some other KDE apps)'s theme will be affected by two different sources, KDE and Qt.

FYI, as explained in https://github.com/nix-community/stylix/pull/1310#issuecomment-3590623674, running KDE with the following declaration seemingly indefinitely freezes KDE:

```nix
stylix.targets.qt.platform = "qtct"`;
```

By default, our NixOS module ensures this is not the case:

> ```nix
>       stylix.targets.qt.platform =
>         [...]
>         else if plasma6.enable && !(gnome.enable || lxqt.enable) then
>           "kde"
> ```
>
> -- [github:nix-community/stylix, `/modules/qt/nixos.nix:29`](https://github.com/nix-community/stylix/blob/e5801bdc21e320ad6693ebd844e0e1e725df4bb8/modules/qt/nixos.nix#L29-L33)
